libs/s3/s3_versioning_common_test_lib.py

```diff
@@ -157,7 +157,6 @@ def check_list_object_versions(s3_ver_test_obj: S3VersioningTestLib,
     :keyword "expected_error": Error message string to verify in case, error is expected
     :keyword "list_params": Dictionary of query parameters to pass List Object Versions call
     """
-    LOG.info("Fetching bucket object versions list")
```

Comment:
Removed this line as it was getting twice : here and in list_object_versions() of  s3_versioning_test_lib.py